.github/workflows/all.testWithRpc.yml

```diff
@@ -6,7 +6,7 @@ on:
 jobs:
   test-node:
     name: testing
-    runs-on: token-studio-linux-medium
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: token-studio-linux-medium
```

Reply:
@nathanklick with that change we have problems to build the sdk..

Reply:
That is a required change that has been made at the organization level. Please revert this change and correct any subsequent build issues.

Reply:
@nathanklick changes done =)